src/Compilers/Core/Portable/Symbols/Attributes/CommonEventWellKnownAttributeData.cs

```diff
@@ -27,5 +27,25 @@ public bool HasSpecialNameAttribute
             }
         }
         #endregion
+
+        #region ExcludeFromCodeCoverageAttribute
+
+        private bool _hasExcludeFromCodeCoverageAttribute;
```

Comment:
> Not sure. We do it for ```SpecialNameAttribute``` for example, which is also only used in emit.

Well, we are not always instrumenting, even when emitting. It feels like a customer shouldn't pay for this feature unconditionally. 

---
In reply to: [91632020](https://github.com/dotnet/roslyn/pull/15720#discussion_r91632020) [](ancestors = 91632020)